doc/examples/applications/plot_haar_extraction_selection_classification.py

```diff
@@ -58,52 +56,56 @@ def extract_feature_image(img, feature_type, feature_coord=None):
                              feature_type=feature_type,
                              feature_coord=feature_coord)
 
-
-###############################################################################
-# We will use a subset of the CBCL which is composed of 100 face images and 100
-# non-face images. Each image has been resized to a ROI of 19 by 19 pixels. We
-# will keep 75 images from each group to train a classifier and check which
-# extracted features are the most salient, and use the remaining 25 from each
-# class to check the performance of the classifier.
+###########################################################################
+# We use a subset of CBCL dataset which is composed of 100 face images and
+# 100 non-face images. Each image has been resized to a ROI of 19 by 19
+# pixels. We select 75 images from each group to train a classifier and
+# determine the most salient features. The remaining 25 images from each
+# class are used to assess the performance of the classifier.
 
 images = lfw_subset()
-# For speed, only extract the two first types of features
+# To speed up the example, extract the two types of features only
 feature_types = ['type-2-x', 'type-2-y']
 
-# Build a computation graph using dask. This allows using multiple CPUs for
-# the computation step
-X = delayed(extract_feature_image(img, feature_types)
-            for img in images)
-# Compute the result using the "processes" dask backend
+# Build a computation graph using dask. This allows to utilize multiple
+# CPU cores later during the actual computation
+X = delayed(extract_feature_image(img, feature_types) for img in images)
+# Compute the result
 t_start = time()
-X = np.array(X.compute(scheduler='processes'))
+if sys.platform.startswith('win'):
+    # To run multiprocessing Dask tasks on Windows, one needs to write
```

Comment:
Is this true? I'll have to test when I boot back up into windows tonight.

Reply:
@hmaarrfk I don't have a Windows machine to cross-validate the fix, but the comment is valid for the Azure Windows VMs (been there, done that :)).